### DISCUSSION TITLE
Preserve empty lines

### DISCUSSION THREAD

**Discussion by N247S** [redhat-developer/vscode-yaml#756] [Q&A] [answered]
I was wondering if there was a way to configure the yalm formatter in such a way that it preserves an x-amount of empty lines (global code-formatting reasons)?

I noticed that several extensions use someting like `Beautify` to define formatting rules, but I cannot find any documentation about something simular for this extension. So I hope someone can point me in the right direction.

Thanks in advance!

**Comment by JPinkney** [comment, marked as answer]
We currently use prettier as the formatter but our formatter code doesn't pick up the prettier config so we are limited to only a few formatter configuring options for now. 

It can't currently preserve an x-amount of empty lines yet but might be possible in the future

**Reply by jgspratt** [reply]
Any chance you'll implement this soon?
